packages/openactive-broker-microservice/app.js

```diff
@@ -1,29 +1,29 @@
-const express = require("express");
-const logger = require("morgan");
-const request = require("request");
-const axios = require("axios");
-const nSQL = require("@nano-sql/core").nSQL;
-const moment = require("moment");
-const config = require("config");
-const criteria = require("@openactive/test-interface-criteria").criteria;
+/* eslint-disable no-console */
```

Comment:
I'm surprised that this is needed if the lint config has `env.node: true` 🤔

Reply:
Ye is strange...